<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>View2DWidget</class>
    <widget class="QWidget" name="View2DWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>960</width>
                <height>640</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>2D Viewer</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,20">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,4,3">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QToolButton" name="shot_btn">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">background-color: rgb(32, 74, 135);
                                            color: rgb(255, 255, 255);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>record</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QToolButton" name="reset_btn">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">background-color: rgb(78, 154, 6);
                                            color: rgb(255, 255, 255);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>rebuild</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="QToolButton" name="hydrogen_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(164, 0, 0);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>revert H</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="pick_layout" stretch="1,3,1">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="pick_label">
                                    <property name="font">
                                        <font>
                                            <pointsize>16</pointsize>
                                        </font>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">color: rgb(0, 0, 0);
                                            background-color: rgb(255, 255, 255);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>selected</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTextEdit" name="pick_edit">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="inputMethodHints">
                                        <set>Qt::ImhNone</set>
                                    </property>
                                    <property name="readOnly">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="expand_layout" stretch="1,1">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QToolButton" name="expand_btn">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">background-color: rgb(196, 160, 0);
                                                    color: rgb(255, 255, 255);
                                                </string>
                                            </property>
                                            <property name="text">
                                                <string>expand</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QToolButton" name="copy_btn">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">background-color: rgb(143, 89, 2);
                                                    color: rgb(255, 255, 255);
                                                </string>
                                            </property>
                                            <property name="text">
                                                <string>copy</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="format_layout" stretch="1,2,2">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="format_label">
                                    <property name="styleSheet">
                                        <string notr="true">color: rgb(0, 0, 0);
                                            background-color: rgb(255, 255, 255);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>format</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="format_box">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="focusPolicy">
                                        <enum>Qt::NoFocus</enum>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">QComboBox {
                                            border : 2px solid gray ;
                                            border-radius: 4px ;
                                            padding : 1px 2px 1px 20px ;
                                            }
                                        </string>
                                    </property>
                                    <item>
                                        <property name="text">
                                            <string>smi</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>pdb</string>
                                        </property>
                                    </item>
                                </widget>
                            </item>
                            <item>
                                <widget class="QToolButton" name="format_btn">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">background-color: rgb(32, 74, 135);
                                            color: rgb(255, 255, 255);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>export</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QWidget" name="container" native="true">
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
